app/models/spree/variant.rb

```diff
@@ -168,63 +167,17 @@ def fees_by_type_for(distributor, order_cycle)
       OpenFoodNetwork::EnterpriseFeeCalculator.new(distributor, order_cycle).fees_by_type_for self
     end
 
-    # returns number of units currently on backorder for this variant.
-    def on_backorder
-      inventory_units.with_state('backordered').size
-    end
-
-    def gross_profit
-      cost_price.nil? ? 0 : (price - cost_price)
-    end
-
     # use deleted? rather than checking the attribute directly. this
     # allows extensions to override deleted? if they want to provide
     # their own definition.
     def deleted?
       deleted_at
     end
 
-    def set_option_value(opt_name, opt_value)
-      # no option values on master
-      return if is_master
```

Comment:
I could have sworn this was the method that was getting called when the various [nil variant migrations](https://github.com/openfoodfoundation/openfoodnetwork/pull/6841) were failing, but I took another look at them and I think I'm misremembering. We're sure it's dead code? I don't see that string `set_option_value` anywhere but here and the spec, so it definitely seems dead. 

https://gifimage.net/wp-content/uploads/2017/11/im-not-dead-yet-gif-1.gif

Reply:
It looks like it was used in an old Spree method in the Spree::Variant class, which we removed.